buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -95,7 +89,8 @@
     private String version;
     private File javaHome;
     private volatile Process esProcess;
-    private final String path;
+    final String path;
+    final LinkedHashMap<String, String> defaultConfig;
```

Comment:
In the spirit of good encapsulation lets make this private and add a method for adding additional config.

Reply:
same as above, not arguing against it, but would leave it for now